src/libraries/System.Net.Http/ref/System.Net.Http.cs

```diff
@@ -144,6 +152,7 @@ public abstract partial class HttpContent : System.IDisposable
         public System.Threading.Tasks.Task<System.IO.Stream> ReadAsStreamAsync(System.Threading.CancellationToken cancellationToken) { throw null; }
         public System.Threading.Tasks.Task<string> ReadAsStringAsync() { throw null; }
         public System.Threading.Tasks.Task<string> ReadAsStringAsync(System.Threading.CancellationToken cancellationToken) { throw null; }
+        protected virtual void SerializeToStream(System.IO.Stream stream, System.Net.TransportContext? context, System.Threading.CancellationToken cancellationToken) { }
```

Comment:
Weren't we going to add ReadAsStream and a virtual CreateContentReadStream sync method as well?

Reply:
Filed API suggestion: https://github.com/dotnet/runtime/issues/37495